test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -195,26 +382,36 @@ test.requestHooks(logger, enrollViaQRcodeMocks)('should be able to enroll via qr
   const enrollOktaVerifyPage = await setup(t);
   await checkA11y(t);
   await t.expect(enrollOktaVerifyPage.getFormTitle()).eql('Set up Okta Verify');
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(true);
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaEmailInstruction()).eql(false);
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaSmsInstruction()).eql(false);
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(true);
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaEmailInstruction()).eql(false);
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaSmsInstruction()).eql(false);
   await t.expect(enrollOktaVerifyPage.hasQRcode()).eql(true);
-  await t.expect(enrollOktaVerifyPage.getQRInstruction()).eql(qrCodeInstruction);
+  const qrInstructionBullet1 = await enrollOktaVerifyPage.getQRInstruction(0);
+  const qrInstructionBullet2 = await enrollOktaVerifyPage.getQRInstruction(1);
+  const qrInstructionBullet3 = await enrollOktaVerifyPage.getQRInstruction(2);
+  await t.expect(qrInstructionBullet1).contains(qrCodeInstruction1);
+  await t.expect(qrInstructionBullet2).contains(qrCodeInstruction2);
+  await t.expect(qrInstructionBullet3).contains(qrCodeInstruction3);
 
   // Verify links
-  await t.expect(await enrollOktaVerifyPage.switchAuthenticatorLinkExists()).ok();
-  await t.expect(enrollOktaVerifyPage.getSwitchAuthenticatorLinkText()).eql('Return to authenticator list');
+  await t.expect(await enrollOktaVerifyPage.returnToAuthenticatorListLinkExists()).ok();
   await t.expect(await enrollOktaVerifyPage.signoutLinkExists()).ok();
 
+  isSuccess = true;
   await t.wait(4000);
-  await t.expect(logger.count(
-    record => record.response.statusCode === 200 &&
-    record.request.url.match(/poll/)
-  )).eql(1);
+  // V3 - higher poll requests ~ 7
```

Comment:
is there a ticket open for this issue that we can include here?

Reply:
I think this is due to the auth.js polling issue which I include here.